l10n_pt_spms/models/product_template.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Dixmit
+# Copyright 2025 NuoBiT - Deniz Gallo <dgallo@nuobit.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = "product.template"
+
+    spms_lot_id = fields.Many2one(
+        "spms.lot",
```

Comment:
```suggestion
        comodel_name="spms.lot",
```
Put this in every field